docs/platform/what-is-tizen/profiles/iot.md

```diff
@@ -0,0 +1,118 @@
+# Tizen IoT
+
+Tizen IoT profile allows you to make various kinds of IoT devices and develop applications for the IoT devices.
+Tizen IoT Headed and Headless binaries support any type of IoT devices based on the Linux kernel.
+
+![iot_building_block](media/iot_building_block.png)
+
+
+## ABB: API building block
+
+Tizen IoT provides a building block pool of components mainly based on the Tizen native API sets.
+
+|Tizen Native API Sets|
+|-----------------|
+|Account Native API Group|
+|Application Framework Native API Group|
+|Base API Group|
+|Content API Group|
+|Context API Group|
+|Location API Group|
+|Machine Learning API Group|
+|Maps API Group|
+|Messaging API Group|
+|Multimedia API Group|
+|Network API Group|
+|Security API Group|
+|Social API Group|
+|System API Group|
+|Telephony API Group|
+|UI API Group|
+|UIX API Group|
+|Web API Group|
+
+
+Custom IoT images can be created by combining the building blocks in various ways according to the needs.
+
+Required packages will be installed based on the dependency related to the specified building blocks.
+
+
+## Types of Building Block Presets
+
+### IoT Core
+- The minimum set of bootable modules that make up the Tizen platform image for IoT devices.
+- The preset contains core components that are required for a basic system image to boot.
+
+### Headless
+- The set of modules that make up the Tizen platform image for headless IoT devices without a display.
+- The preset contains IoT core presets with other components specifically required for headless image.
+- Example of specific components for headless includes Audio moduldes, Bluetooth modules, and etc.
+
+### Headed
+- The set of modules that make up the Tizen platform image for headed IoT devices with a display.
+- The preset contains IoT core presets with other components specifically required for headed image.
+- Example of specific components for headed includes Notification API, graphic modules, and etc.
+
+
+## MIC Imager Creator
+
+MIC is an image creator that is used to create images for Tizen. 
+
+The tool offers three major functions as follows:
+- creating an image in a specific format
+- chrooting into an image
+- coverting an image to another format
+
+For more information about the tool and its functions, refer to [MIC Image Creator](https://docs.tizen.org/platform/reference/mic/mic-overview/).
+
+In this document, we will focus on creating the image using MIC.
+
+To create a custom Tizen IoT image, 
+
+0) Check prerequisites
+    - OS: Ubuntu 14.04 LTS or higher
+    - Git Build System(GBS) and Image Creator(MIC) for Tizen developers
+        ```shell
+        $ echo "deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_16.04/ /" | sudo tee /etc/apt/sources.list.d/tizen.list
+        $ sudo apt-get update
+        $ sudo apt-get install gbs mic
+        ```
+
+1) Download a kistart(.ks) file from the [Tizen download site](http://download.tizen.org/snapshots/tizen/unified/). 
```

Comment:
Global: Is it Kickstart file? Please clarify

Download a Kickstart (.ks) file from the [Tizen download site]

Reply:
done ;)